coins.json

```diff
@@ -1260,5 +1260,26 @@
             "clientPublic": "", 
             "clientDocs": "https://polkadot.js.org/api/substrate/rpc.html"
         }
+    },
+    {
+        "id": "filecoin",
+        "name": "Filecoin",
+        "symbol": "FIL",
+        "decimals": 8,
+        "blockchain": "Filecoin",
+        "derivationPath": "m/44'/12345'/0'/0'",
+        "curve": "secp256k1",
+        "publicKeyType": "secp256k1Extended",
+        "explorer": {
+            "url": "https://??",
```

Comment:
Explorer link should be filled

Reply:
There is no explorer with "transaction by ID" and "transactions by address" lookups available yet.